Headers/P7_Trace.h

```diff
@@ -79,73 +79,73 @@ enum eP7Trace_Level
                                                                      __FILE__,\
                                                                      __FUNCTION__,\
                                                                      i_pFormat,\
-                                                                     __VA_ARGS__)
+                                                                     ##__VA_ARGS__)
 
 ////////////////////////////////////////////////////////////////////////////////
 #define P7_QTRACE(i_wID, i_hModule, i_pFormat, ...)    P7_DELIVER(i_wID,\
                                                                   EP7TRACE_LEVEL_TRACE,\
                                                                   i_hModule,\
                                                                   i_pFormat,\
-                                                                  __VA_ARGS__)
+                                                                  ##__VA_ARGS__)
 #define P7_TRACE(i_hModule, i_pFormat, ...)            P7_QTRACE(0,\
                                                                  i_hModule,\
                                                                  i_pFormat,\
-                                                                 __VA_ARGS__)
+                                                                 ##__VA_ARGS__)
```

Comment:
`##__VA_ARGS__` is a GCC extension which is, AFAIK, is not supported in MSVC.

In future, C++2a will solve this problem with something like `foo(bar, __VA_OPT__(,) __VA_ARGS__`)

Reply:
Solved 12e1dc0ad3a6c06be9e7115aebbc2950b650e6ff